### PR TITLE
[1LP][RFR] Migration UI refactor

### PR DESCRIPTION
**Pull request by Yadnyawalkya**
{{pytest: cfme/tests/v2v/test_v2v_migrations_ui.py --use-provider rhv43 --use-provider vsphere67-ims --provider-limit 2 -vvvv}}

- Removed sorting and host tag testcase since this aren't necessary.
- Moving host refresh test to somewhere else I might raise new PR for that which includes tier4/chaos tests

Reviewers, better to review with `cfme/tests/v2v/test_v2v_migrations_ui.py` instead of diff.

### CODE REVIEW

**Comment by Yadnyawalkya** [comment]
@sshveta  @digitronik  you can start reviewing. 5.11 PRT failing with breadcrumb issue (https://bugzilla.redhat.com/show_bug.cgi?id=1703744).

File: cfme/tests/v2v/test_v2v_migrations_ui.py [hunk 1]
```diff
@@ -13,395 +15,303 @@
...
     """
-    # TODO: This test case does not support update
-    # as update is not a supported feature for mapping.
-    infrastructure_mapping_collection = appliance.collections.v2v_mappings
-    mapping = infrastructure_mapping_collection.create(form_data_single_datastore)
-    view = navigate_to(infrastructure_mapping_collection, 'All')
-    if appliance.version >= '5.10':  # As mapping could be on any page due to pagination
-        infrastructure_mapping_collection.find_mapping(mapping)  # We need to find mapping first
-    soft_assert(mapping.name in view.infra_mapping_list.read())
+    # TODO: Add edit map test after infra map list refactor
```
Comment (sshveta) [inline cfme/tests/v2v/test_v2v_migrations_ui.py]: Remove this TODO now

File: cfme/v2v/infrastructure_mapping.py [hunk 2]
```diff
@@ -69,6 +69,11 @@ def after_fill(self, was_change):
             if not self.parent.add_mapping.disabled:
                 self.parent.add_mapping.click()
 
+        @property
+        def is_displayed(self):
+            return (self.source.is_displayed and self.source.is_displayed and
```
[the remark this answers is not in the document]
Comment (Yadnyawalkya) [reply cfme/v2v/infrastructure_mapping.py]: crap! updated.

File: cfme/tests/v2v/test_v2v_migrations_ui.py [hunk 3]
```diff
@@ -13,395 +15,322 @@
...
-        assert mapping.name not in view.infra_mapping_list.read()
-    except NoSuchElementException:
-        # meaning there was only one mapping that is deleted, list is empty
-        pass
+    # Test3: Source cluster from UI
+    soft_assert(mapping.clusters[0].sources[0].format() in
+                mapping_list.get_map_source_clusters(mapping.name)[0])
 
+    # Test4: Target cluster from UI
+    soft_assert(mapping.clusters[0].targets[0].format() in
```
Comment (mshriver) [inline cfme/tests/v2v/test_v2v_migrations_ui.py]: What's the purpose of these empty `.format()` calls?

Comment (Yadnyawalkya) [reply cfme/tests/v2v/test_v2v_migrations_ui.py]: if you see value of `mapping.clusters[0].targets[0]` it is in `partial_match` format.. so to convert it into normal data we need this `.format()`

**Pull request merged**